mapactionpy_controller/map_doc.py

```diff
@@ -93,8 +93,10 @@ def to_xml(self):
         location = SubElement(mapdata, 'location')
         location.text = str(self.mapdata.location)
 
-        access = SubElement(mapdata, 'themes')
-        access.themes = str(self.mapdata.themes)
+        themes = SubElement(mapdata, 'themes')
+        for setTheme in self.mapdata.themes:
+            theme = SubElement(themes, 'theme')
+            theme.text = setTheme
```

Comment:
For nearly all the other value in the method, the value is explictly cast to `str. I've not tried it, but is a cast required here?